guides/tutorials/postgres-database-nodejs/windows.md

```diff
@@ -5,7 +5,7 @@
 1. Download and install Visual Studio Community Edition 2022
 1. Install git for windows
 1. Download rust for windows [here](https://www.rust-lang.org/en-US/install.html)
```

Comment:
This is supposed to be numbered list, right? If yes, we will have to update the list numbers (2, 3, 4)

Reply:
In markdown, numbered lists are added automatically if you give all items a 1. prefix. It's handy for when you change the order / add / remove items, as you don't have to update the numbers